package.json

```diff
@@ -20,6 +20,7 @@
     "infusion": "2.0.0",
     "kettle": "1.6.4",
     "sjrk-storytelling": "https://github.com/waharnum/sjrk-storyTelling#FLOE-525",
+    "couch-config": "https://github.com/waharnum/couch-config#SJRK-6",
```

Comment:
This dependency name should be `sjrk-couch-config` instead, which is the name in the `package.json` file (I should update repo name as well to avoid confusion)